worker/jobTypes/githubJob.js

```diff
@@ -2,6 +2,8 @@ const fs = require('fs-extra');
 const workerUtils = require('../utils/utils');
 const simpleGit = require('simple-git/promise');
 const request = require('request');
+const logger = require('../utils/logger');
+const S3Publish = require('./S3Publish');
```

Comment:
We have avoided putting a binding to S3 in the Github job because it implies a dependency on S3 which should not be the case for these operations. I didn't see a reference to this const in this PR though so do we need it?

Reply:
We do not need logger no - I'm sorry i should have closed this PR after that dummy spec I wrote for this (and then reopened it when officially ready). I was still in the process of cleaning everything up!

Reply:
Yes, please close the PR and reopen. Let me know when ready to review (I'm assuming it's still not ready because these lines are still in the code).